lib/result-view.coffee

```diff
@@ -227,12 +226,9 @@ transformimeJupyter.consoleTextTransform.mimetype = [
 ]
 
 transform = transformime.createTransform [
-    transformimeJupyter.PDFTransform,
     transformime.ImageTransformer,
     transformimeJupyter.SVGTransform,
     transformimeJupyter.consoleTextTransform,
-    transformimeJupyter.LaTeXTransform,
     MarkdownTransform,
     transformime.HTMLTransformer,
-    transformimeJupyter.ScriptTransform
```

Comment:
Do these get removed because of how we end up using transformime within Hydrogen, rather than relying on a transform based on the full bundle?